backend/api/chatLink/utils/validateChannel.test.ts

```diff
@@ -0,0 +1,63 @@
+import channelValid, { CHANNEL_STATE } from './validateChannel';
+import db from "../../../db";
+
+jest.mock('../../../db');
+
+describe('channelValid', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    // test if channel is provided otherwise throw new Error("channel - required param");
+    it('should throw an error is the channel is not provided', async () => {
+        await expect(channelValid('')).rejects.toThrow("channel - required param");
+    });
+
+    it('should return NOT_FOUND if the channel does not exist in the db', async () => {
+        const findOneFromDBMock = db.findOneFromDB as jest.MockedFunction<typeof db.findOneFromDB>;
+        findOneFromDBMock.mockResolvedValueOnce(null);
+        // WHere is the above being used?
+        const result = await channelValid('nonexistent_channel');
+        expect(result.valid).toBe(false);
+        expect(result.state).toBe(CHANNEL_STATE.NOT_FOUND);
```

Comment:
One more assertion that can be useful: 
`expect(db.findOneFromDB).toBeCalledWith({ hash: 'nonexistent_channel' });`

Reply:
Thanks for the review, will work on this.